elastic/beats
# enrich container id from process id

## Type of change
- Enhancement

## What does this PR do?
This PR enables add process metadata to enrich container id in Kubernetes environment by cgroup using process id.

## Why is it important?
Include important metadata of the container id in the Kubernetes environment, which beats don't have this ability yet.

## Checklist




## Author's Checklist



## How to test this PR locally



Deploy Auditbeat in minikube with the include_cid set to true. Deploy an sshd server in minikube. After the Auditbeat is running, ssh into the sshd server, which will trigger an Auditbeat event. See if the event coming from the sshd container is enriched with container id `cid`

## Related issues

Closes elastic/beats#14967 


- 

## Use cases



## Screenshots

![image](https://user-images.githubusercontent.com/24740574/73407802-1a873500-42af-11ea-8d2a-d690268e5540.png)



## Logs
